### PR TITLE
Update associating-an-email-with-your-gpg-key.md

### LINKED ISSUE #2722: missing step in "Associating an email with your gpg key"
### What article on docs.github.com is affected?
[Associating an email with your gpg key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/associating-an-email-with-your-gpg-key)

### What part(s) of the article would you like to see updated?
This misses an important step.
To actually save the changes to the pgp key, you have to type `pgp> save` into the console.
The current wording makes is sound like confirming the changes with `o` saves the changes, which is not the case.

### Additional information
I think the article would benefit from a clarification, so it is easier to follow. 

Edit: See #2723 for proposed changes

### PR DESCRIPTION
**Pull request by layaxx**
add necessary step to "associating an email with your gpg key"

### Why:
This is currently worded ambiguous and misses an important step.
See Issue #2722 

### What's being changed:
- improve wording to clarify that changes are confirmed but not saved at this point
- add a step to actually save the changes to the pgp key

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Review by jace4411 — APPROVED** [review]

**Comment by janiceilene** [comment]
Thanks for your patience! Our small team is working our way through reviewing all of the amazing contributions ✨

File: content/github/authenticating-to-github/associating-an-email-with-your-gpg-key.md [hunk 1]
```diff
@@ -32,14 +32,18 @@ If you're using a GPG key that matches your committer identity and your verified
...
   ```
-7. Enter `O` to save your selections.
+7. Enter `O` to confirm your selections.
 8. Enter your key's passphrase.
-9. Enter `gpg --armor --export GPG key ID`, substituting in the GPG key ID you'd like to use. In the following example, the GPG key ID is `3AA5C34371567BD2`:
+9. Enter `gpg> save` to save the changes
+  ```shell
+  $ gpg> save
+  ```
+10. Enter `gpg --armor --export GPG key ID`, substituting in the GPG key ID you'd like to use. In the following example, the GPG key ID is `3AA5C34371567BD2`:
```
Comment (Mesho098) [inline content/github/authenticating-to-github/associating-an-email-with-your-gpg-key.md:41]: 10. Enter `gpg --armor --export GPG key ID`, substituting in the GPG key ID you'd like to use. In the following example, the GPG key ID is `3AA5C34371567BD2`:

**Review by Mesho098 — CHANGES_REQUESTED** [review]

**Review by felicitymay — APPROVED** [review]
Hi @layaxx 👋🏻 

Thank you for raising this PR to fix the omission that you spotted. I checked your instructions and they work perfectly. Your changes look great and I'll update your branch and merge it now. 🎉

**Review by mqdmm — COMMENTED** [review]
سلام دوستان عزیز

**Review by mqdmm — COMMENTED** [review]
سلام دوستان عزیز @

**Review by mqdmm — COMMENTED** [review]
دوباره سلام به روی ماه نشسته شما.خیلی دوستتان دارم ❤️🤗❤️🧚🧚

**Pull request merged**